app/containers/List/ListPicker.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+import { Text } from 'react-native';
+
+import { TActionSheetOptionsItem, useActionSheet } from '../ActionSheet';
+import { CustomIcon } from '../CustomIcon';
+import * as List from './index';
+import I18n from '../../i18n';
+import { useTheme } from '../../theme';
+import sharedStyles from '../../views/Styles';
+
+type TOption = { label: string; value: string; second?: string };
+type TOptions = { [key: string]: TOption[] };
+
+interface IListPicker {
+	title: string;
+	testID: string;
+	value?: string;
+	options: TOptions;
+	preference: string;
+	onChangeValue: (pref: string, param: { [key: string]: string }, onError: () => void) => void;
+}
+
+const ListPicker = ({ preference, options, value, title, testID, onChangeValue }: IListPicker) => {
```

Comment:
This is a generic component.
What's `preference` on this context?
It should be some other name.